perun-rpc/src/main/java/cz/metacentrum/perun/rpc/ApiCaller.java

```diff
@@ -475,4 +475,40 @@ public void stateChangingCheck() throws RpcException {
 	public Object call(String managerName, String methodName, Deserializer parms) throws PerunException {
 		return PerunManager.call(managerName, methodName, this, parms);
 	}
+
+	/**
+	 * Returns a view of the portion of this list between the specified fromIndex, inclusive, and toIndex, inclusive.
+	 *
+	 * If list of objects is null or empty, it will return empty list as sublist.
+	 * If fromIndex is lower than 0, it will be set to 0.
+	 * If toIndex is bigger than size of an input array, it will be set to the size of an array.
+	 * If fromIndex and toIndex are same or fromIndex is bigger than toIndex, it will return empty array.
```

Comment:
First part of the sentence is not true. If I pass eg. 5/5 and sourcing list contains 6+ groups, it will return group at the index 5 (6th group).